scripts/src/main/resources/scripts/functions

```diff
@@ -817,13 +828,31 @@ function doReplaceExtractedSkipSingleFolder() {
   local source_dir="${1}"
   local target_dir="${2}"
   local backup_dir="${3}"
+  local root_dir="${1}"
   shift 3
+  local files
+  local anz_files
+  local dir="${1}"
   if [ "${#*}" = 1 ] && [ "${1%.app}" = "${1}" ] && [ -d "${1}" ]
   then
     doDebug "Changing source directory from ${source_dir} to ${1} due to single folder."
-    source_dir="${1}"
+    files="$(find "${dir}" -maxdepth 1 -mindepth 1 -exec basename {} \;)"
+    anz_files="$(echo "${files}" | wc -w)"
+    while [ "${anz_files}" == "1" ]
+    do
+      if [ -d "${dir}/${files}" ] && [ "${files}" != "bin" ] && [ "${files%.app}" = "${files}" ]
+      then
+        dir="${dir}/${files}"
+      else
+        break
+      fi
+      files="$(find "${dir}" -maxdepth 1 -mindepth 1 -exec basename {} \;)"
+      anz_files="$(echo "${files}" | wc -w)"
+    done
+    source_dir="${dir}"
```

Comment:
In case you changed `source_dir` the above debug message would be incorrect.
Can you simply move it down here?

Reply:
Yes, of course. I move it to the end and adapt the variables